### DISCUSSION TITLE
Service Bindings

### DISCUSSION THREAD

**Discussion by rafaeltuelho** [redhat-developer/odo#6282] [Q&A] [answered]
Hi I've got a couple of questions about odo and Service Bindings.
Does odo only support Binding Services for services backed by Operators? If not, is there any example showing how to create a binding to a svc (eg. Database) not backed by Operator?
Asking this because sometimes the developer wants to test its app in a cluster where the operator may not be available for him, and he does not have permission to install an operator in the cluster.

**Comment by feloy** [comment, marked as answer]
Hi Rafael,

The command `odo add binding` is used to add information about a binding into the Devfile. This command only supports services backed by operators.

But it is still possible to add the binding information manually. Here is an example of adding such a binding in the Devfile manually, for an external service: https://odo.dev/blog/binding-external-service-with-odo-v3 .

**Reply by rafaeltuelho** [reply]
Hi @feloy 
Thanks for sharing your blog post. That's exactly what I was looking for.
